Specs/DateTime/Italian/DurationParser.json

```diff
@@ -432,20 +449,21 @@
           "PastResolution": {
             "duration": "86400"
           }
-        }
+        },
+        "Start": 17,
+        "Length": 13
       }
     ]
   },
   {
-    "Input": "Andrò via per l'intero weekend",
+    "Input": "Andrò via per l'intera settimana",
```

Comment:
"l'intero weekend" is not a common expression even if informal?
If it's somewhat common, we should support it.

Reply:
'l'intero weekend' (the whole weekend) is not supported in English either. 
Also this test case seems a bad translation of the case 'I'll leave for full week' since the expected Timex refers to a week duration.
Should we add a test case for l'intero weekend and skip it?

Reply:
Please do. Thanks.